.github/workflows/backup-20250905-114531/ai-models-validation.yml

```
@@ -0,0 +1,72 @@
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2.2.2
+        with:
+          version: 8.6.0
+          run_install: false
+      - name: Get pnpm store directory
+        shell: bash
+        run: |
+          echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+
+        run: pnpm install --frozen-lockfile
```

actionlint (1.7.7)

key "run" is duplicated in element of "steps" section. previously defined at line:34,col:9

(syntax-check)